includes/widgets/image-carousel.php

```diff
@@ -174,6 +174,116 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'navigation_left_icon',
```

Comment:
Please never use explicit direction, because on RTL it may be on the right.
(here and next control)
```suggestion
			'navigation_prev_icon',
```

Reply:
@matipojo Done